Python/compile.c

```diff
@@ -1205,16 +1205,11 @@ _PyCompile_OptimizationLevel(compiler *c)
 
 int
 _PyCompile_IsInteractive(compiler *c)
```

Comment:
What do you think about renaming this to something like `_PyCompile_IsInteractiveTopLevel` for clarity?